npm/pkg/dataplane/policies/policy_windows.go

```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/Microsoft/hcsshim/hcn"
 )
 
+const (
+	blockRulePriotity = 3000
```

Comment:
May be beneficial to put comments why they are `3000` and `222` for code maintenance.